toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -54,6 +62,12 @@ while read -r package || [ -n "$package" ]; do
     install_one_toolchain_rpm "$package"
 done < "$packages"
 
+$bldtracker \
```

Comment:
This is repeated a few times (and more if we add to the loop). Maybe worth defining a function that only takes a single argument (since the other arguments are constant).